app/routes/admin.py

```
@@ -9662,6 +9579,48 @@ def help_support():
     selected_class_label = selected_option["label"] if selected_option else None
 
+    class_scope_options = [
+        {
+            'class_id': ce_row.class_id,
+            'join_code': get_display_join_code(ce_row.class_id),
+            'label': ce_row.display_name or get_display_join_code(ce_row.class_id),
+        }
+        for ce_row in teacher_user_class_rows
+    ]
+
+    def _support_report_views(issues):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_support_report_views`

(ANN202)